.travis.yml

```diff
@@ -3,7 +3,14 @@ language: ruby
 rvm:
   - "2.3.7"
 
-dist: precise
+dist: trusty
```

Comment:
I had to upgrade to allow `libatk-bridge2.0-0` to be installed otherwise Cypress dies.